specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -358,6 +374,25 @@ not required to upload.
 
   @doc("Configuration for quick load test")
   optionalLoadTestConfig?: OptionalLoadTestConfig;
+
+  @added(APIVersions.v2024_03_01_preview)
+  @doc("Locust test configuration, in case kind is Locust")
+  locustConfiguration?: LocustConfiguration;
+
+  @added(APIVersions.v2024_03_01_preview)
+  @doc("Region distribution configuration for the load test.")
+  regionalLoadTestConfig?: RegionalConfiguration[];
+}
+
+@doc("Region distribution configuration for the load test.")
+model RegionalConfiguration {
+  @doc("""
+  The number of engine instances to execute load test in specified region. Supported values are in range of 1-400.
+""")
+  engineInstances: int32;
+
+  @doc("Name of region")
+  region: string;
```

Comment:
Could you address meeting comment?
https://github.com/Azure/azure-rest-api-specs/issues/28376#issuecomment-2037496122